arcgis-ios-sdk-samples/Layers/Display subtype feature layer/DisplaySubtypeFeatureLayerViewController.swift

```diff
@@ -54,10 +54,10 @@ class DisplaySubtypeFeatureLayerViewController: UIViewController {
     }
     
     func makeMap() -> AGSMap {
+        AGSAuthenticationManager.shared().delegate = self
```

Comment:
This seems like a very odd place for this. Could it go somewhere else, like maybe immediately before the call to `makeMap()`?